src/cedict-service/cedict.js

```diff
@@ -331,7 +331,8 @@ export default class Cedict {
     const requests = this._configuration.data.chunks.map(chunk => this._loadJson(`${this._configuration.data.URI}/${chunk}`))
     return Promise.all(requests).then(chunks => {
       let meta = chunks[0].metadata // eslint-disable-line prefer-const
-      delete this.cedict.meta.chunkNumber
+      meta.cedict = chunks[0].cedictMeta
```

Comment:
this looks strange to me.  are you intentially inserting the cedict meta into the main metadata objecty? why?

Reply:
I've noticed that we do not store CEDICT metadata everywhere. It can be useful for us in the future for things like copyright notices and so on. So I decided to fix this and thought we could add it to the metadata object because it's the same type of meta data, but related to CEDICT (so it will be within the `cedict` property). That will allow us to keep all metadata together in one IndexedDB store, without the need to create a separate store just for CEDICT meta.

What do you think about that? If you think it does not make sense I will remove it.

Reply:
I think it's a good idea, It just seems weird to me as written - at first glance I thought you were overwriting the variable you had just set. But I think it's okay. Maybe just add a comment. Thanks.

Reply:
Adding a comment is a great ides, updated code with it.